packages/debug/src/node/debug-service-impl.ts

```diff
@@ -71,7 +71,7 @@ export class DebugServiceImpl implements DebugService {
         // TODO: Support dynamic debug configurations through Theia extensions?
         return {};
     }
-    fetchDynamicDebugConfiguration(name: string, type: string): Promise<DebugConfiguration | undefined> {
+    fetchDynamicDebugConfiguration(name: string, type: string, folder: string | undefined): Promise<DebugConfiguration | undefined> {
```

Comment:
Same as before:

```suggestion
    fetchDynamicDebugConfiguration(name: string, type: string, folder?: string): Promise<DebugConfiguration | undefined> {
```

Reply:
All the remarks have been addressed in the latest commit !!
Thanks Vince !!